build.psm1

```diff
@@ -188,6 +191,26 @@ function Start-PSBuild {
     # verify we have all tools in place to do the build
     $precheck = precheck 'dotnet' "Build dependency 'dotnet' not found in PATH. Run Start-PSBootstrap. Also see: https://dotnet.github.io/getting-started/"
 
+    $dotnetIntalledVersion = (dotnet --version)
+    If ( $dotnetIntalledVersion -ne $dotnetCLIRequiredVersion ) {
+        Write-Warning @"
+The currently installed .Net Core version is not the required version.
```

Comment:
> The currently installed .Net Core version

Should be `The currently installed .NET Command Line Tools is not the required version.`